apps/infra-local/Makefile

```
@@ -0,0 +1,88 @@
+# BoxLite-based infra-local orchestrator
+#
+# All targets are thin wrappers around `python -m boxlite_local`. The Python
+# CLI is the source of truth — see `python -m boxlite_local --help` for the
+# full surface area. This Makefile exists for muscle memory + tab completion.
+
+PY ?= python
+
+.PHONY: help up down ps doctor logs wipe itest test install migrate e2e itest-all \
+        stack-build stack-up stack-down stack-restart stack-status stack-logs stack-reset stack-reset-hard stack-nuke \
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)